apps/angular/permissions/src/app/user.model.ts

```diff
@@ -1,5 +1,9 @@
-export type Role = 'MANAGER' | 'WRITER' | 'READER' | 'CLIENT';
-
+export enum Role {
+  MANAGER = 'MANAGER',
+  WRITER = 'WRITER',
+  READER = 'READER',
+  CLIENT = 'CLIENT',
+}
```

Comment:
you should prefer union type over enums. That's a question of preference, but at least you don't have to decalre the enum inside your template